data/locations.yaml

```diff
@@ -118,7 +118,7 @@ events:
     organizers:
       - Johanna Bayer, Kelly Garner
     contact:
-      - TBD
+      - johanna.bayer@orygen.org.au
     website: TDB
```

Comment:
Can we get this changed to https://brainhack-aus.github.io/global2021/ too? thanks!

Reply:
Done